xilem/examples/to_do_mvc.rs

```diff
@@ -84,11 +89,15 @@ fn app_logic(task_list: &mut TaskList) -> impl WidgetView<Edit<TaskList>> + use<
                     },
                 )
                 .text_size(16.);
-                let delete_button_label = label("Delete").text_size(16.);
-                let delete_button = button(delete_button_label, move |data: &mut TaskList| {
+                let delete_button = text_button("Delete", move |data: &mut TaskList| {
                     data.tasks.remove(i);
-                });
-                Some(flex_row((checkbox, delete_button)))
+                })
+                .padding(2.0);
```

Comment:
A padding of 2 looks so cramped. I'd do 5 minimum.

Reply:
The original todo mvc actually displays all items as a table, but doing so with borders without having a double-border is fiddly. But you're right, current look is worse. Let's add padding.